lldb/include/lldb/Target/Process.h

```diff
@@ -2294,6 +2294,9 @@ void PruneThreadPlans();
 
   lldb::ThreadPlanSP FindDetachedPlanExplainingStop(Thread &thread, Event *event_ptr);
 
```

Comment:
I know you just moved the function around, but it would be nice if the method had a Doxygen comment.

Reply:
✅